lucien-ios/Lending/LendABookViewController.swift

```diff
@@ -0,0 +1,153 @@
+    }
+
+    // MARK: - Private Instance Methods
+
+    private func configureTableView() {
+        tableView.register(lendingCell, forCellReuseIdentifier: "userCell")
+
+        tableView.rx.itemSelected.subscribe(onNext: { [weak self] indexPath in
+            self?.nextButton.isEnabled = true
+            self?.nextButton.tintColor = LucienTheme.dark
+            self?.selectedIndex = indexPath.row
+            self?.tableView.reloadData()
+        }) >>> disposeBag
+
+        viewModel.users.asObservable().bind(to: tableView.rx.items(
+            cellIdentifier: "userCell",
+            cellType: LendingTableViewCell.self)
+        ) { [weak self] row, user, cell in
+            cell.nameLabel.text = "\(user.firstName) \(user.lastName)"
+            cell.emailLabel.text = user.email
+            if row  == self?.selectedIndex {
+                cell.accessoryType = .checkmark
+                cell.accessoryView = UIImageView(image: UIImage(named: "checkmark"))
+            } else {
+                cell.accessoryType = .none
+            }
+        } >>> disposeBag
+    }
+
+    private func configureNavigationController() {
+        setNavBarBackground()
+        setNavBarTitle()
+        setNavBarBackButton()
+        setNavBarNextButton()
+    }
+
+    private func setNavBarBackground() {
+        navigationController?.navigationBar.isTranslucent = true
+        navigationController?.navigationBar.barStyle = .default
+        navigationController?.navigationBar.barTintColor = LucienTheme.white
+        navigationController?.navigationBar.setValue(true, forKey: "hidesShadow")
+    }
+
+    private func setNavBarTitle() {
+        navigationController?.viewControllers[0].title = "Lend Book"
+        navigationController?.navigationBar.titleTextAttributes = [NSAttributedStringKey.font: LucienTheme.Fonts.permanentMarkerRegular(size: 30) ?? UIFont()]
+    }
+
+    private func setNavBarBackButton() {
+        let backButton = UIBarButtonItem()
+        backButton.image = UIImage(named: "navBackButton")
+        backButton.style = .plain
+        backButton.rx.tap.subscribeNext { [weak self] in
+            self?.navigationController?.popViewController(animated: true)
+        } >>> disposeBag
+        backButton.tintColor = UIColor.black
+        navigationItem.leftBarButtonItem = backButton
+    }
+
+    private func setNavBarNextButton() {
+        nextButton = UIBarButtonItem(title: "Next", style: .plain, target: self, action: nil)
+        nextButton.setTitleTextAttributes([NSAttributedStringKey.font: LucienTheme.Fonts.muliSemiBold(size: 17) ?? UIFont()], for: .normal)
+        nextButton.tintColor = LucienTheme.finishButtonGrey
+        nextButton.isEnabled = false
+        // TODO: Add implementation for next button
+        navigationItem.rightBarButtonItem = nextButton
+    }
+
+    // MARK: - UITableViewDelegate
+    
+    func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
+        return 68
```

Comment:
I'd like to encourage everyone to use constraints to determine cell heights these days instead of defining them in the delegate like this. https://developer.apple.com/library/content/documentation/UserExperience/Conceptual/AutolayoutPG/WorkingwithSelf-SizingTableViewCells.html